disnake/types/appinfo.py

```diff
@@ -73,3 +73,19 @@ class PartialAppInfo(BaseAppInfo, total=False):
 class PartialGatewayAppInfo(TypedDict):
     id: Snowflake
     flags: int
+
+
+class EditAppInfo(TypedDict, total=False):
+    custom_install_url: Optional[str]
+    description: str
+    role_connections_verification_url: Optional[str]
+    install_params: Optional[InstallParams]
+    integration_types_config: Dict[Literal[0, 1], ApplicationIntegrationTypeConfiguration]
+    flags: int
+    icon: Optional[str]
+    cover_image: Optional[str]
+    interactions_endpoint_url: Optional[str]
+    tags: Optional[List[str]]
+    event_webhooks_url: Optional[str]
+    event_webhooks_status: Literal[1, 2]
```

Comment:
This literal should be separate at the top of the file (`AppInfo.event_webhooks_status` also specifies `str` which isn't quite correct). I know it can't be set to `3` manually, but we probably don't have to be that accurate here.